src/main/java/frc/robot/Constants.java

```diff
@@ -306,5 +306,10 @@ public static final class ClimberConstants {
     public static final boolean kLeftMotorReversed = false;
     public static final boolean kRightMotorReversed = false;
     public static final PID kPID = new PID(0.5, 0.0, 0.0);
+    public static final double kGearRatio = 3.0;
+    public static final double kVelocityConversionFactor = ((0.012 / 60) / kGearRatio);
+    public static final double kPositionConversionFactor = 0.012 / kGearRatio;
```

Comment:
What is the 0.012? This needs to be explained in a comment. And the fact that it shows up twice suggests that the constants need to be refactored such that 0.012 only shows up in one place.